ha/core/system_health/system_health.py

```diff
@@ -85,6 +85,22 @@ def get_node_status(self, node_id, **kwargs):
         """
         get node status method. This method is for returning a status of a node.
         """
+        try:
+            # Prepare key and read the health value.
+            key = self._prepare_key(const.COMPONENTS.NODE_MAP.value, node_id=node_id)
+            node_map_val = self.healthmanager.get_key(key)
+            if node_map_val is not None:
+                node_map_dict = eval(node_map_val)
+                key = self._prepare_key(component='node', cluster_id=node_map_dict['cluster_id'],
+                                        site_id=node_map_dict['site_id'], rack_id=node_map_dict['rack_id'],
+                                        storageset_id=node_map_dict['storageset_id'], node_id=self.node_id, **kwargs)
+                return self.healthmanager.get_key(key)
+            else:
+                return
+
+        except Exception as e:
+            Log.error(f"Failed reading status for component: node with Error: {e}")
+            raise HaSystemHealthException("Failed reading status")
```

Comment:
Include node id in the exception string.

Reply:
Added node_id in exception string.